### DISCUSSION TITLE
Import to image to costume but not to new costume!

### DISCUSSION THREAD

**Discussion by TheFirstOrangeScratcher7777** [ScratchAddons/ScratchAddons#5856] [Ideas]
It is like import image to costume that exsist, I want it because in Scratch 2 it very helps me in makeing costumes! I  think it is long ago be in Scratch 3 but I do not remember. :\

**Comment by Samq64** [comment]
Related: #3788

**Reply by TheFirstOrangeScratcher7777** [reply]
I do not see that, can you send a photo or just it is on Scratch 2?

**Reply by Samq64** [reply]
There is a pull request open that allows pasting image from the clipboard onto an existing costume, but as far as I know it doesn't add a button to upload one onto that costume.
